packages/checkout/sdk-sample-app/src/components/CheckConnection.tsx

```diff
@@ -36,6 +36,7 @@ export default function CheckConnection(props: CheckConnectionProps) {
       const resp = await checkout.checkIsWalletConnected({
         provider,
       });
+      console.log('Connection result', resp);
```

Comment:
do we want to log this? 👀

Reply:
I thought it was helpful just because the UI is not really refreshing clearly.. I also figured it was fine as it's really our internal SDK sample app.. but happy to remove if you prefer @mimi-imtbl ?